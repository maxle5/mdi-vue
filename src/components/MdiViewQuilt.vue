
<template>
  <svg :width="size" :height="size" :viewBox="viewBox">
      <path d="M10,5V11H21V5M16,18H21V12H16M4,18H9V5H4M10,18H15V12H10V18Z" style="fill: currentColor" />
  </svg>
</template>

<script lang="ts">
  import { defineComponent } from 'vue';

  export default defineComponent({
      name: "MdiViewQuilt",
      props: {
          size: { 
              type: Number, 
              default: 24 
          }
      },
      computed: {
        viewBox() {
          return `0 0 ${this.size} ${this.size}`;
        }
      }
  })
</script>
