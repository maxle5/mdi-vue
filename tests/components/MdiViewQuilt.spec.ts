
import { mount } from "@vue/test-utils";
import { expect, test } from "vitest";
import MdiViewQuilt from "../../src/components/MdiViewQuilt.vue";

test("MdiViewQuilt snapshot", () => {
  const wrapper = mount(MdiViewQuilt, {});
  expect(wrapper.html()).toMatchSnapshot();
});
