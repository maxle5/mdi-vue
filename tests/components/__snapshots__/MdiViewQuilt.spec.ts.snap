// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MdiViewQuilt snapshot 1`] = `
"<svg width="24" height="24" viewBox="0 0 24 24">
  <path d="M10,5V11H21V5M16,18H21V12H16M4,18H9V5H4M10,18H15V12H10V18Z" style="fill: currentColor;"></path>
</svg>"
`;
